src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static final SessionFactory instance = initSessionFactory();
```

Comment:
pay attention to the point related to HibernateUtil class and constructor 
https://mate-academy.github.io/jv-program-common-mistakes/hibernate/configuration/configuration_checklist